src/index.html

```diff
@@ -22,6 +25,395 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#">
+        <img
+          class="header__logo"
+          src="images/logo.png"
+          alt="Moyo logo"
+        />
+      </a>
+
+      <nav class="header__nav">
+        <ul class="header__nav_list">
```

Comment:
Class names should be separated by a hyphen, not an underscore. Rename `header__nav_list` to `header__nav-list` to follow the BEM naming convention.